db/mem_db.go

```diff
@@ -37,7 +37,8 @@ func (db *MemDB) Get(key []byte) []byte {
 	defer db.mtx.Unlock()
 	key = nonNilBytes(key)
 
-	return db.db[string(key)]
+	value := db.db[string(key)]
```

Comment:
why not just return? copy?

Reply:
No, just so it's easier to add print statements for debugging.  Or, I had to anyways, and this is an artifact.  I sometimes leave these around because it's convenient, and it doesn't seem like bad form.  How do you feel about it?

Reply:
I've used delve before, that helps.  Print statements are still good too.  Have you tried Microsoft's Golang tooling?  I dunno if they have a debugger integrated.  What do you use?